ui/tests/helpers/clients/client-count-selectors.ts

```diff
@@ -68,11 +68,12 @@ export const CLIENT_COUNT = {
   upgradeWarning: '[data-test-clients-upgrade-warning]',
 };
 
-export const CHART_ELEMENTS = {
-  entityClientDataBars: '[data-test-group="entity_clients"]',
```

Comment:
these weren't used anywhere, plan is to migrate nested selectors above ☝️  into this block